src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -107,7 +107,7 @@ const FileCard: React.FC<FileCardProps> = ({ file, removeFile, toggleLicenseModa
                     <RadioGroup
                       aria-label="version"
                       {...field}
-                      onChange={(event) => setFieldValue(field.name, event.target.value === 'published')}>
+                      onChange={(event) => form?.setFieldValue(field.name, event.target.value === 'published')}>
```

Comment:
kan vi ikke hente form fra nærmeste parent Field her, så vi slipper å sende det ned som prop? samme for resten nedover her

Reply:
jo, det kan vi forsåvidt! det er jo mye bedre 👍